zubhub_frontend/zubhub/src/views/edit_profile/EditProfile.jsx

```diff
@@ -363,63 +368,186 @@ function EditProfile(props) {
-                        onBlur={props.handleBlur}
-                        endAdornment={
-                          <InputAdornment position="end">
-                            <IconButton
-                              aria-label="toggle password visibility"
-                              onClick={() =>
-                                handleSetState(handleClickShowPassword(state))
-                              }
-                              onMouseDown={handleMouseDownPassword}
-                              edge="end"
-                            >
-                              {show_password ? (
-                                <Visibility />
-                              ) : (
-                                <VisibilityOff />
+                        <InputLabel
+                          className={classes.customLabelStyle}
+                          htmlFor="password"
+                        >
+                          {t('editProfile.inputs.password.label')}
+                        </InputLabel>
+                        <OutlinedInput
+                          className={classes.customInputStyle}
+                          id="password"
+                          name="password"
+                          type={show_current_password ? 'text' : 'password'}
+                          onChange={props.handleChange}
+                          onBlur={props.handleBlur}
+                          endAdornment={
+                            <InputAdornment position="end">
+                              <IconButton
+                                aria-label="toggle password visibility"
+                                onClick={() =>
+                                  handleSetState(handleClickShowPassword(state, 'show_current_password'))
+                                }
+                                onMouseDown={handleMouseDownPassword}
+                                edge="end"
+                              >
+                                {show_current_password ? (
+                                  <Visibility />
+                                ) : (
+                                  <VisibilityOff />
+                                )}
+                              </IconButton>
+                            </InputAdornment>
+                          }
+                          label={t('editProfile.inputs.password.label')}
+                        />
+                        <FormHelperText
+                          className={classes.fieldHelperTextStyle}
+                          error
+                        >
+                          {(props.status && props.status['password']) ||
+                            (props.errors['password'] &&
+                              <Trans t={t} i18nextKey="editProfile.inputs.changePassword.error">
+                                <ErrorIcon fontSize="small" /> {{errorMessage: props.errors['password']}}
+                              </Trans>
```

Comment:
using the `Trans` component from `react-i18next` to improve the field error message with Icons